pull_request_template.md

```diff
@@ -74,15 +74,21 @@
 	- Should be named `Contents`, not `Table of Contents`.
 	- Should be the first section in the list.
 	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
+    - Must not feature `Contributing`, `License` or `Footnotes` sections.
```

Comment:
Use tab-indentation. Applies elsewhere too.

Reply:
Oops. Sorry. Fixed in: https://github.com/sindresorhus/awesome/pull/1860/commits/fcd2f66af11677468edf272017280c3aaab5f099